Assets/Creatures/impl/Cerebrian/Cerebrax/Cerebrax.yml

```diff
@@ -0,0 +1,6 @@
+speed: 0.5
```

Comment:
I do think this is pretty cool, as far as how we define characters base attributes

Since these are small right now, it's probably not worth bringing up

But maybe if this grows in an unbounded way, we'd want to compose it in some way?

Like `hasBaseAnimations: true` or something?

Reply:
yeah that's a good point we can keep that in mind